File: Task3/cpp/factorial.cpp
Q. To find factorial of a number

Sol.
//here is iterative solution to find factorial and can be improved by dp
//including header files
#include<bits/stdc++.h>
using namespace std;

// function to find factorial of given number 
unsigned int factorial(unsigned int n) 
{ 
    int res = 1, i; 
    for (i = 2; i <= n; i++) 
        res *= i; 
    return res; 
} 
  
// Driver code 
int main() 
{ 
    int num = 5; 
    cout << "Factorial of " << num << " is " << factorial(num) << endl; 
    return 0; 
} 

File: Task3/javascript/factorial.js
//to find factoial of a number 10
function factorial(n){
  let answer = 1;
  if (n == 0 || n == 1){
    return answer;
  }else{
    for(var i = n; i >= 1; i--){
      answer = answer * i;
    }
    return answer;
  }  
}
let n = 10;
answer = factorial(n)
console.log("The factorial of " + n + " is " + answer);
